tests/docker/test_cli.py

```diff
@@ -0,0 +1,63 @@
+def test_generate_dataset_in_container(capsys, run_in_container):
+    run_in_container(
+        [
+            "generate_dataset",
+            "--help",
+        ]
+    )
+
+    captured = capsys.readouterr()
```

Comment:
Using `capsys` (TIL) here is neat, but I feel like it would be better to fix `run_fg` to return its output rather than printing it:
https://github.com/opensafely-core/databuilder/blob/db1a904676d3cdbd943e149df6f268a3a15897fb/tests/lib/docker.py#L48-L49